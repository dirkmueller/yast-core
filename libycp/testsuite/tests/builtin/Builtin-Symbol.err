Parsed:
----------------------------------------------------------------------
"** tosymbol **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
tosymbol ("test")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
tosymbol (69)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
tosymbol (nil)
----------------------------------------------------------------------
[Interpreter] tests/builtin/Builtin-Symbol.ycp:15 Argument (nil) to tosymbol(...) is nil
