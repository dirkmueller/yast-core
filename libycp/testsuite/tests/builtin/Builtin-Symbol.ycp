
# ---------------------------------------------------------
#
#  Filename:	Builtin-Symbol.ycp
#
#  Purpose:	test cases for all symbol builtins
#
# ---------------------------------------------------------


("** tosymbol **")

(tosymbol ("test"))
(tosymbol (69))
(tosymbol (nil))

