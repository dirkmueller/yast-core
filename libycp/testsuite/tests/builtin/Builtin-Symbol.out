("** tosymbol **")
(`test)
(`69)
(nil)
